dynamiq/nodes/agents/base.py

```diff
@@ -298,6 +299,8 @@ class Agent(Node):
     role: str | None = ""
     _prompt_blocks: dict[str, str] = PrivateAttr(default_factory=dict)
     _prompt_variables: dict[str, Any] = PrivateAttr(default_factory=dict)
+    _mcp_servers: list[Node] = PrivateAttr(default_factory=list)
+    _exclude_tools_ids: list[str] = PrivateAttr(default_factory=list)
```

Comment:
Let use here more clear naming, like `_mcp_servers_tools` or smth like this.

Reply:
It seems my comment was unclear, _mcp_servers_tools it's about renaming `_exclude_tools_ids`. `mcp_servers` was OK
```suggestion
    _mcp_servers: list[MCPServerAdapter] = PrivateAttr(default_factory=list)
    _msp_servers_tools_ids: list[str] = PrivateAttr(default_factory=list)
```